code/modules/events/vent_clog.dm

```diff
@@ -26,10 +28,11 @@
 
 /datum/round_event/vent_clog/tick()
 	if(activeFor % interval == 0)
-		var/obj/vent = pick_n_take(vents)
+		var/obj/machinery/atmospherics/components/unary/vent = pick_n_take(vents)
+		while(vent && vent.welded)
+			vent = pick_n_take(vents)

```

Comment:
err, so why is there a while() here now? we're  clogging all the vents all at once?